tests/testsuite/bench.rs

```diff
@@ -444,6 +444,7 @@ fn bench_with_lib_dep() {
             name = "foo"
             version = "0.0.1"
             authors = []
+            autobins = true
```

Comment:
Where `autoXXX = true` was added to the test suite could the files be reorganized a bit to not trigger the warning or need the key?

Reply:
let me have a look.